lib/logstash/plugin_mixins/jdbc.rb

```diff
@@ -18,10 +18,11 @@ def self.included(base)
 
   public
   def setup_jdbc_config
-    # JDBC driver library path to third party driver library.
+    # JDBC driver library path to third party driver library. In case of multiple libraries being
+    # required you can pass them separated by a comma.
     #
     # If not provided, Plugin will look for the driver class in the Logstash Java classpath.
-    config :jdbc_driver_library, :validate => :path
+    config :jdbc_driver_library, :validate => :string
```

Comment:
@purbon I think changing to :array is still compatible. Here's my thinking:

* `:path` is basically `:string` with an additional validation step to verify the path exists
* `:string` is a single "string"
* `:array` can be a single "string" in the config, or `[ "multiple", "values" ]`
* When an `:array` is given a single "string" like `jdbc_driver_library => "foo"` it is the same as if you had a single-element array: `jdbc_driver_library => [ "foo" ]`

So if we change validate to `:array` as @suyograo proposed, we still are backwards compatible, but make it hopefully more obvious how to specify multiple libraries to load.

Thoughts on using `:validate => :array` ?